Story #12674: OPS cleanup since migration to API GW.

GiooDev opened the pull request:
## Description

Cleanup useless code since migration to API GW.

* Fix start & stop playbooks
* Remove useless playbooks or protect unexpected execution
* Update certificate generation for UI as client-external
* Cleanup unused vars in environments/ directory
* Add a dedicated playbook to deploy frontend apps
* Add a parameter to define custom assets (logos)
* Fix API GW consul's registration
* Add a playbook to easily reload certificates in database

## Type de changement

* Ansiblerie
* Refactorisation de code
* PKI

## Contributeur

* VAS (Vitam Accessible en Service)

GiooDev committed:
Story #12674: OPS cleanup since migration to API GW.

GiooDev commented on deployment/environments/group_vars/all/vitamui_vars.yml:
```diff
@@ -54,50 +54,34 @@ vitamui_defaults:
     has_lang_selection: true
     has_site_selection: false
+  ingest:
+    vitamui_component: "ui-ingest"
```
Voir pour retirer complètement ça car si c'est paramétré ça ne pourra plus fonctionner.
Et/ou voir pour retirer les groupes associés et gérer un status enabled pour chacun de ces composants pour permettre de choisir lesquels on souhaite installer ou pas.

vitam-devops commented:
![Logo](https://cdn.ast.checkmarx.net/integrations/logo/Checkmarx.png)
**Checkmarx One – Scan Summary & Details** – [3baaab45\-c6d5\-451f\-9579\-5e788b28ef4c](https://eu.ast.checkmarx.net/projects/3f06ff53-d733-40ac-a6fb-49e415cb793f/scans?id=3baaab45-c6d5-451f-9579-5e788b28ef4c&branch=story_12674_cleanup_migration_api_gw)
### New Issues
Severity|Issue|Source File / Package|Checkmarx Insight
---|---|---|---
![HIGH](https://cdn.ast.checkmarx.net/integrations/severity/High.png)|Passwords And Secrets \- Generic Password|/vitamui\_vars\.yml: [237](https://github.com/ProgrammeVitam/vitam-ui/blob/story_12674_cleanup_migration_api_gw//deployment/environments/group_vars/all/vitamui_vars.yml#L237)|Query to find passwords and secrets in infrastructure code.
![MEDIUM](https://cdn.ast.checkmarx.net/integrations/severity/Medium.png)|[Cx366abb53-9fde](https://devhub.checkmarx.com/cve-details/Cx366abb53-9fde)|Npm\-es5\-ext\-0\.10\.62|[Vulnerable Package](https://eu.ast.checkmarx.net/sca/#/projects/3f06ff53-d733-40ac-a6fb-49e415cb793f/reports/3baaab45-c6d5-451f-9579-5e788b28ef4c/vulnerabilities/Cx366abb53-9fde%3ANpm-es5-ext-0.10.62/vulnerabilityDetails)

### Fixed Issues
Severity|Issue|Source File / Package
---|---|---
![HIGH](https://cdn.ast.checkmarx.net/integrations/severity/High.png)|Passwords And Secrets \- Generic Password|/vitamui\_vars\.yml: [302](https://github.com/ProgrammeVitam/vitam-ui/blob/story_12674_cleanup_migration_api_gw//deployment/environments/group_vars/all/vitamui_vars.yml#L302)

bbenaissa commented on deployment/roles/nginx_webapp/templates/frontend/vhost.conf.j2:
Vérifier l'ip d'écoute du vhost, limiter à localhost pas ok si déporté sur un autre rp.

server_name utile ou pas ?
à tester: retirer le server_name

GiooDev replied:
À priori le server_name ne semble pas poser de problème à retirer.
Par contre, si on passe à localhost il faut updater l'upstream aussi.

GiooDev committed:
Story #12734: Listen to ip_service for vhost ui.

GiooDev committed:
Story #12674: Add confirmation on uninstall playbooks to avoid execution errors.

GiooDev committed:
Story #12674: Remove useless assets.

GiooDev committed:
Story #12674: Cleanup useless mongo scripts as they are deleted later.

GiooDev committed:
Story #12674: Provide a custom playbook to reload certificates if recreated.

Can also be used during migration if needed.
Remove oldest mongo scripts to load certs and add a new one to do them
all.

GiooDev committed:
Story #12674: Update API GW configuration if deployed on consul server.

GiooDev committed:
Story #12674: Update certificate generation for UI as client-external.

GiooDev committed:
Story #12674: Fix start/stop playbooks.
